Bumped version in archetype simple

svenkubiak opened the pull request:
Every time I create a new project I do the same thing in the simple archetype pom.xml. So, I thought why not change the archetype itself. 

I bumped most of the versions and removed some blank lines.

Commit:
bumped version in archetype simple

buildhive commented:
[Ninja Web Framework » ninja #102](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/102/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

raphaelbauer commented:
Hi Sven,

nice :) 

There are two things that come to my mind:
1) Your editor changed from tabs to spaces (or vice versa). So the diff is huge. Could you change that back to the original convention? That way we can clearly see what is going on :)
2) TinyMCE and bootstrap are imho directly referenced in the application ftl.html files (src="/assets/bootstrap/3.0.0...") or so... I bet the application does not properly style now.. Can you make sure that the editor and boostrap gets loaded properly (you can also write a testcase against it...)

Thanks :)

Best,

Raphael

Commit:
reverted changes to pom.xml in archetype

Commit:
bumped webjar and referenced libraries versions

svenkubiak commented:
Sorry about the formatting. Updated my IDE this week and forgot to change the damn spacing. I have updated the templates now as well. There was one odd thing though. JQuery 1.9.0 is reference in the template, but it isn't in the pom.xml and as far as I can see it didn't come with TinyMCE or Bootstrap. So I added the latest version of JQuery.

buildhive commented:
[Ninja Web Framework » ninja #108](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/108/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

buildhive commented:
[Ninja Web Framework » ninja #109](https://buildhive.cloudbees.com/job/ninjaframework/job/ninja/109/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

raphaelbauer commented:
perfect... many thanks for updating the archetype :+1:
